### PR TITLE
feat(image): add image loading events

### PR DESCRIPTION
**Pull request by etasia79**
I need to get the image load end event on a static image.
I added the 3 corresponding events (imageloadstart, imageloadend, imageloaderror), see here :+1: http://openlayers.org/en/latest/apidoc/module-ol_source_ImageWMS-ImageWMS.html

(The documentation only exits for ImageWMS, but it works for ImageStatic too).

@Neonox31 / @davinkevin / @Yakoust

### CODE REVIEW

**Comment by davinkevin** [comment]
The linting failed:

```

ERROR: /home/travis/build/quentin-ol/ngx-openlayers/src/app/color-select-hover/color-select-hover.component.ts[21, 60]: Delete `:`
```

File: projects/ngx-openlayers/src/lib/sources/imagestatic.component.ts [hunk 1]
```diff
@@ -30,12 +30,25 @@ export class SourceImageStaticComponent extends SourceComponent implements OnIni
...
+  @Output()
+  onImageLoadEnd: EventEmitter<source.ImageEvent>;
+  @Output()
+  onImageLoadError: EventEmitter<source.ImageEvent>;
+
   constructor(@Host() layer: LayerImageComponent) {
     super(layer);
+    this.onImageLoadStart = new EventEmitter<source.ImageEvent>();
+    this.onImageLoadEnd = new EventEmitter<source.ImageEvent>();
+    this.onImageLoadError = new EventEmitter<source.ImageEvent>();
```
Comment (davinkevin) [inline projects/ngx-openlayers/src/lib/sources/imagestatic.component.ts]: @Yakoust I still think we should not initialize this here but directly in the field. Can we start to do this for new elements and move old ones one by one (my favorite boy scout strategy).

Comment (Yakoust) [reply projects/ngx-openlayers/src/lib/sources/imagestatic.component.ts]: Yes agree with that :+1:

Comment (etasia79) [reply projects/ngx-openlayers/src/lib/sources/imagestatic.component.ts]: I've just fixed it and pushed.

File: projects/ngx-openlayers/src/lib/sources/imagewms.component.ts [hunk 2]
```diff
@@ -36,13 +46,26 @@ export class SourceImageWMSComponent extends SourceComponent implements OnChange
...
+  @Output()
+  onImageLoadStart: EventEmitter<source.ImageEvent>;
+  @Output()
+  onImageLoadEnd: EventEmitter<source.ImageEvent>;
+  @Output()
+  onImageLoadError: EventEmitter<source.ImageEvent>;
+
   constructor(@Host() layer: LayerImageComponent) {
     super(layer);
+    this.onImageLoadStart = new EventEmitter<source.ImageEvent>();
```
Comment (Yakoust) [inline projects/ngx-openlayers/src/lib/sources/imagewms.component.ts]: As for image static, we should initialize those directly in the field

Comment (etasia79) [reply projects/ngx-openlayers/src/lib/sources/imagewms.component.ts]: ouch, forgot this one, sorry...

**Comment by Yakoust** [comment]
:+1:  LGTM @Neonox31 ? @davinkevin

**Pull request merged**